salt/_modules/caasp_filters.py

```diff
@@ -40,3 +40,15 @@ def get_max(seq):
     # TODO: use the builtin filter (https://docs.saltstack.com/en/latest/topics/jinja/index.html#max)
     #       once we Salt>2017.7.0
     return max(seq)
+
+
+def is_admin_node():
```

Comment:
Maybe it would make more sense to move this function to [`caasp_nodes.py`](https://github.com/kubic-project/salt/blob/master/salt/_modules/caasp_nodes.py), as there are many functions for getting nodes by their roles... The function couold then be renamed to `is_admin`, so we would call it with `salt.caasp_nodes.is_admin(...)`

Reply:
Done